traffic_ops/traffic_ops_golang/types/types.go

```diff
@@ -211,3 +215,324 @@ func deleteQuery() string {
 WHERE id=:id`
 	return query
 }
+
+// Get [Version :V5]
```

Comment:
Could use a bit more description like: `GetV5 will retrieve a list of types`. And the function name has to match the start of the comment.